src/tests/Microsoft.PowerFx.Core.Tests/LocalizationTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.Localization;
+using Xunit;
+using System.Reflection;
+using System.Linq;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class LocalizationTests
```

Comment:
>LocalizationTests [](http://example.com/codeflow?start=17&length=17)

? git merge issue?

Reply:
This and the corresponding .resx* changes should definitely be in a separate PR.

Reply:
We are adding ErrExpectedIdentifierArg_Name message

Reply:
This test verifies we are not forgetting any message (which was the case), hence a couple of renames